book/src/11_advanced_component.md

```diff
@@ -0,0 +1,140 @@
+# Advanced strategies for components
+
+So, now we have a fairly good grasp of the basics of Specs, so this is 
+where we start experimenting with more advanced patterns!
```

Comment:
This sentence doesn't flow well for me. Maybe it could be rewritten to something like: "So now that we have a fairly good grasp on the basics of Specs, it's time that we start to experiment with more advanced patterns!"